.gitignore

```diff
@@ -12,6 +12,7 @@ phpcs.xml
 yarn.lock
 docker-compose.override.yml
 /wordpress
+.vscode
```

Comment:
You can create a global gitignore file instead to so you don't have to include IDE specific ignores to each project:
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

We would have to keep a long list of IDE specific files for each WordPress project instead :)

Reply:
Done, thanks 👍